src/hooks/useConnectCurrentVenue.ts

```diff
@@ -14,6 +14,23 @@ const useConnectCurrentVenue = () => {
       storeAs: "currentVenue",
     },
   ]);
+
+  useFirestoreConnect([
+    {
+      collection: "venues",
+      doc: venueId,
+      subcollections: [{ collection: "subvenues" }],
+      storeAs: "currentSubVenues",
+    },
+  ]);
+
+  useFirestoreConnect([
+    {
+      collection: "venues",
+      where: [["parentId", "==", venueId]],
+      storeAs: "currentChildVenues",
```

Comment:
Too much complexity, let's scrap this parent/child thing, and only have subvenues.

In SparkleTown, different venues are "down the street from one another".

Reply:
Ah, do you want to get rid of the `subvenue` collection within a venue and store all the information in the linked `Venue` document?

Reply:
The opposite; ie. if there are sub-spaces within a venue they live in that venue.